consensus/ibft/consensus_backend.go

```diff
@@ -103,11 +104,12 @@ func (i *backendIBFT) ID() []byte {
 	return i.currentSigner.Address().Bytes()
 }
 
-func (i *backendIBFT) MaximumFaultyNodes() uint64 {
+// Q: This is not being used in go-ibft
+func (i *backendIBFT) MaximumFaulty() uint64 {
 	return uint64(CalcMaxFaultyNodes(i.currentValidators))
 }
```

Comment:
Remove it?
```suggestion
```

Reply:
Done https://github.com/0xPolygon/polygon-edge/pull/837/commits/f7f7b532085845605c43b23f607ac4dde32275c8